submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/SubmissionApiTest.kt

```diff
@@ -73,6 +81,48 @@ internal class SubmissionApiTest(private val tempFolder: TemporaryFolder) : Base
             assertThat(submissionRepository.getByAccNo("SimpleAcc1")).isEqualTo(submission)
         }
 
+        @Test
+        fun `empty accNo`() {
+            val submission = tsv {
+                line("Submission")
+                line("Title", "Empty AccNo")
+            }.toString()
+
+            assertThat(webClient.submitSingle(submission, SubmissionFormat.TSV)).isSuccessful()
+            assertThat(submissionRepository.getByAccNo("S-BSST0")).isEqualTo(
+                submission("S-BSST0") {
+                    title = "Empty AccNo"
+                }
+            )
+        }
+
+        @Test
+        fun `empty accNo with parent`() {
+            val project = tsv {
+                line("Submission", "A-Project")
+                line("AccNoTemplate", "!{S-APR,}")
+                line()
+
```

Comment:
any particular reason for this line break

Reply:
I mean the single `line()` already represents a line break so not need to add another line break. Do I am missing something?

Reply:
That's just to make the pagetab more readable. We're doing the same in all the other cases and I think it looks good and makes clear at a single glance, where the line breaks are:

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/factory/TsvSubmissionFactory.kt#L13

Reply:
it looks for me as a double line but is a matter of test. Completely subjective.

Reply:
Ok, maybe we should change these and all the others in a separated PR, just to be consistent.